bittensor_cli/src/commands/stake/children_hotkeys.py

```diff
@@ -160,6 +159,8 @@ async def set_childkey_take_extrinsic(
     # Decrypt coldkey.
     try:
         wallet.unlock_coldkey()
```

Comment:
Do we wanna use the unlocker here as well?

Reply:
Yeah, I didn't want to because the unlocker prints to err_console, but these functions expect a success and message. This was easier than adding another arg for like `print_to_console: bool` for the unlocker function, imo. But I'll change it.